custom_components/bhyve/binary_sensor.py

```diff
@@ -89,3 +90,41 @@ def _on_ws_data(self, data):
 
     def _should_handle_event(self, event_name, data):
         return event_name in [EVENT_FS_ALARM]
+
+    
+class BHyveTemperatureAlert(BHyveDeviceEntity):
+
+    def __init__(self, hass, bhyve, device):
+        name = "{} temperature alert".format(device.get("name"))
+        super().__init__(hass, bhyve, device, name, "alert")
+
+    def _setup(self, device):
+        self._available = device.get("is_connected", False)
+        self._state = self._parse_status(device.get("status", {}))
+    
+    def _parse_status(self, status):
+        """Convert BHyve alarm status to entity value."""
+        return "on" if status.get("temp_alarm_status") == "high_temp_alarm" else "off"
```

Comment:
As far as I can tell, the device also allows setting a low temperature alert.

I'm unsure the right way to model this in Home Assistant. I can see one of two options:
1. One `TemperatureBinarySensor` where `state() -> "on"` if `temp < low || temp > high`. Entity attributes would have the appropriate attributes for `temp_threshold_low`, `temp_threshold_low_exceeded`, `temp_threshold_high` and `temp_threshold_high_exceeded`
2. Two binary sensors: `LowTemperatureBinarySensor` and `HighTemperatureBinarySensor` (should be self explanatory)

Reply:
I’m going to go with your first suggestion. The idea being that this entity is in alarm. This is how bHyve handles this in their app.

Reply:
This should do it…
return "on" if "alarm" in status.get("temp_alarm_status") else "off"